pyaedt/modules/SolveSetup.py

```diff
@@ -979,7 +979,7 @@ def add_sweep(self, sweepname=None, sweeptype="Interpolating"):
 
         Returns
         -------
-        SweeHFSS3DLayout
+        :classs:`pyaedt.modules.SetupTemplates.SweepHFSS3DLayout`
```

Comment:
```suggestion
        :class:`pyaedt.modules.SetupTemplates.SweepHFSS3DLayout`
```
That's my bad, Kathy. There were three 's' and it made the style check fail.

Reply:
No worries. I assume all typos are mine!